zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
This one is OK for the virtio_exp branch to avoid to fork the zephyrproject-rtoss/open-amp and zephyrproject-rtos/libmetal repos.
 Please, could you add the information in commit subject